src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -491,27 +491,38 @@ export class ShelleyWallet extends Wallet implements WalletInterface {
     return addressedUtxos
   }
 
-  async getLegacyAddressedUtxos() {
-    const utxos = await this.fetchUTXOs()
-    const addressedUtxos = utxos.map((utxo: RawUtxo): AddressedUtxo => {
-      const addressing = this.getAddressing(utxo.receiver)
-
-      return {
-        ...utxo,
-        addressing,
-      }
-    })
-
-    return addressedUtxos
-  }
-
   getDelegationStatus() {
     if (!this.transactionCache) throw new Error('invalid wallet state')
     if (this.rewardAddressHex == null) throw new Error('reward address is null')
     const certsForKey = this.transactionCache.perRewardAddressCertificates[this.rewardAddressHex]
     return Promise.resolve(getDelegationStatus(this.rewardAddressHex, certsForKey))
   }
 
+  async getStakingInfo(): Promise<StakingInfo> {
```

Comment:
promote getStakingInfo from utility function to wallet method
moved from `StakePoolInfos.tsx`